test/serializer/additional-functions/ArrayConcat.js

```diff
@@ -1,4 +1,4 @@
-
+// expected FatalError: PP0001
```

Comment:
Wait what this used to work and now it doesn't? Or is what's going on that this diagnostics arises internally, but is then put under the carpet because the entire `.concat` call is made abstract? if so, then this should be reflected in this expected-error assertion.

Reply:
Talking to Chris, it seems to be the case that there's a mismatch between the intention of the test runner and the Prepack cli: The test runner happily declares success for this test case, even though it produces a fatal error, and the Prepack cli will terminate with an error-exit code. Please file an issue for that. And we should address that before marking otherwise passing test cases with producing `expected FatalError`.